### PR TITLE
Unable to convert Imagenet-VID to TFRecords for TF/lstm_object_detection

### PR DESCRIPTION
**Pull request by evgps**
Dear @wakanda-ai and @KanaSukita  I have found that you have succeeded in running code from https://github.com/tensorflow/models/issues/6253 .
Can you please provide more instructions on how to use your convertor and which settings in https://github.com/tensorflow/models/blob/master/research/lstm_object_detection/configs/lstm_ssd_mobilenet_v1_imagenet.config are correct for usage with your code.

### CODE REVIEW

**Comment by KanaSukita** [comment]
Hi@evgps , the command I use for the converter is:

python vid_2015_to_tfrecord.py --root_dir=/path/to/VID2015 --output_path=/path/to/output

It should generate tfrecords containing sequential data with length of 4, which is the same in the config. And in the config, you need to change the input path to your tfrecords.

BTW, optimization for MobilenetV2 is updated recently, you may try that. However, still I couldn't train a valid model using the newest config, so I doubt that the way I convert the tfrecord might be wrong.

**Comment by KanaSukita** [comment]
Hi  @evgps the command I use for the converter is:

python vid_2015_to_tfrecord.py --root_dir=/path/to/VID2015 --output_path=/path/to/output

It should generate tfrecords containing sequential data with length of 4, which is the same in the config. And in the config, you need to change the input path to your tfrecords.

BTW, optimization for MobilenetV2 is updated recently, you may try that. However, still I couldn't train a valid model using the newest config, so I doubt that the way I convert the tfrecord might be wrong.

**Comment by wangpichao** [comment]
> Hi @evgps the command I use for the converter is:
> 
> python vid_2015_to_tfrecord.py --root_dir=/path/to/VID2015 --output_path=/path/to/output
> 
> It should generate tfrecords containing sequential data with length of 4, which is the same in the config. And in the config, you need to change the input path to your tfrecords.
> 
> BTW, optimization for MobilenetV2 is updated recently, you may try that. However, still I couldn't train a valid model using the newest config, so I doubt that the way I convert the tfrecord might be wrong.

@KanaSukita Thanks for all the efforts to run this project. When I used python vid_2015_to_tfrecord.py --root_dir=/path/to/VID2015 --output_path=/path/to/output, I will generate 9 tfrecords, but in the config, the input_path: '/data/lstm_detection/tfrecords/test.tfrecord' redirected to 1 tfrecord. How did you modified this line? Thanks.

**Comment by chicleee** [comment]
Hello, Dear @evgps and @KanaSukita ，thanks for your method。^_^
Do you have successed in running this whole code? and how did it perform?

**Comment by wangpichao** [comment]
@KanaSukita How to set the lab_map_path? Or what is it? How to generate it? Any help would be appreciated. 
train_input_reader: {
  shuffle_buffer_size: 32
  queue_capacity: 12
  prefetch_size: 12
  min_after_dequeue: 4
  label_map_path: "path/to/label_map"
  external_input_reader {
    [lstm_object_detection.protos.GoogleInputReader.google_input_reader] {
      tf_record_video_input_reader: {
        input_path: './research/lstm_object_detection/inputs/tfrecords/VID_2015-0.tfrecord'
        data_type: TF_SEQUENCE_EXAMPLE
        video_length: 4
      }
    }
  }
}

**Comment by KanaSukita** [comment]
@wangpichao I write the vid_label_map.pbtxt myself, something like below:

item {
  name: "n02691156"
  id: 1
  display_name: "airplane"
}
item {
  name: "n02419796"
  id: 2
  display_name: "antelope"
}
item {
  name: "n02131653"
  id: 3
  display_name: "bear"
}
...

**Comment by wangpichao** [comment]
> @wangpichao I write the vid_label_map.pbtxt myself, something like below:
> 
> item {
> name: "n02691156"
> id: 1
> display_name: "airplane"
> }
> item {
> name: "n02419796"
> id: 2
> display_name: "antelope"
> }
> item {
> name: "n02131653"
> id: 3
> display_name: "bear"
> }
> ...

@KanaSukita thanks. Till now, are you still working on this project?

**Comment by KanaSukita** [comment]
@wangpichao No, I couldn't train a valid model even if I use the training data for evaluation. But I am willing to try again if there is any upate in the official repo.

**Comment by wangpichao** [comment]
> @wangpichao No, I couldn't train a valid model even if I use the training data for evaluation. But I am willing to try again if there is any upate in the official repo.

@KanaSukita I also tried several times and with no valid models.
